packages/contracts-bedrock/scripts/DeployConfig.s.sol

```diff
@@ -103,7 +103,10 @@ contract DeployConfig is Script {
         requiredProtocolVersion = stdJson.readUint(_json, "$.requiredProtocolVersion");
         recommendedProtocolVersion = stdJson.readUint(_json, "$.recommendedProtocolVersion");
 
-        if (block.chainid == Chains.LocalDevnet || block.chainid == Chains.GethDevnet) {
+        if (
```

Comment:
This is annoying to keep up to date, perhaps we want `onlyDevnetOrTestnet` as a library

Reply:
Yeah this would be nice to have.

Reply:
Probably worth creating a nice-to-have ticket for this.